src/app/components/ServicePage.jsx

```diff
@@ -0,0 +1,76 @@
+import Image from "next/image";
+import { FaArrowRight, FaArrowLeft, FaExternalLinkAlt } from "react-icons/fa";
+
+const Button = ({ text, className, icon }) => (
+  <button
+    className={`bg-gray-800 text-white px-4 py-2 rounded hover:bg-gray-700 transition flex items-center space-x-2 ${className}`}
+  >
+    <span>{text}</span>
+    {icon && <span>{icon}</span>}
+  </button>
+);
+
+const ServiceSection = ({ title, description, highlights }) => (
+  <div className="border border-about-bg p-6 rounded-lg space-y-4">
+    <div className="flex items-center space-x-2">
+      <h2 className="text-2xl font-semibold flex items-center space-x-2">
+        <span>{title}</span>
+        <FaExternalLinkAlt className="text-blue-400" />
+      </h2>
+    </div>
+    <p>{description}</p>
+    <div className="space-y-2">
+      {highlights.map((highlight, index) => (
+        <div key={index} className="flex items-center space-x-2">
+          <span className="text-blue-400">✦</span>
+          <p>{highlight}</p>
+        </div>
+      ))}
+    </div>
+  </div>
+);
+
+export default function Services() {
```

Comment:
Name of the component doesn't match the name of the file
Also I'm not sure that it should be called ServicePage as it's not a page